draft-sharabayko-mops-srt.md

```diff
@@ -648,7 +754,7 @@ For more details refer to {{handshake-messages}}.
 |                                                               |
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 ~~~~
-{: #keymaterial-extension-structure title="Key Material Extension structure"}
+{: #fig-km-msg title="Key Material Message structure"}
 
 S ( ): 1 bit. Value: {0}  
 : This is a fixed-width field that is a remnant from the header of a previous design.
```

Comment:
Should that "previous design" be identified?

Reply:
Maybe better to say it is a reserved field with value of 0. 🤔

Reply:
Replaced with "This is a fixed-width field that is reserved for future usage."